e2e-tests/ukef/cypress/e2e/journeys/portal/submit-AIN-deal-creates-tasks-in-tfm/submit-AIN-deal-creates-tasks-in-tfm.spec.js

```diff
@@ -72,9 +72,7 @@ context('Portal to TFM deal submission', () => {
     cy.clearCookie('_csrf');
     cy.getCookies().should('be.empty');
 
-    cy.forceVisit(TFM_URL);
-
-    cy.tfmLogin(BUSINESS_SUPPORT_USER_1);
+    cy.mockTfmLogin(BUSINESS_SUPPORT_USER_1, TFM_URL);
```

Comment:
Could the TFM login command have a default `TFM_URL`, save passing it in

Reply:
This was solved some time ago. No need for URL param as it is always the same.